### PR TITLE
fix(module_utils.js): add missing node_modules in Lambda envs

### PR DESCRIPTION
**Pull request by ranrib**

### CODE REVIEW

File: src/consts.js [hunk 1]
```diff
@@ -34,6 +34,9 @@ module.exports.DEFAULT_SAMPLE_RATE = 1;
 // Key name to inject epsagon correlation ID
 module.exports.EPSAGON_HEADER = 'epsagon-trace-id';
 
+// In some cases we manually add the Lambda node_modules path, where it is not found by default
+module.exports.LAMBDA_DEFAULT_NODE_MODULES_PATH = '/var/task/node_modules';
```
Comment (haimrait) [inline src/consts.js:38]: the user can change it if he wants? with the environment variable?

File: src/events/module_utils.js [hunk 2]
```diff
@@ -53,15 +55,20 @@ module.exports.getModules = function getModules(id) {
...
             autoNodePaths = getAllNodeModulesPaths(rootFolder);
         }
+        utils.debugLog('Found the following paths', autoNodePaths);
         autoNodePaths.forEach((nodePath) => {
             if (!searchPaths.includes(nodePath)) {
                 searchPaths.push(nodePath);
             }
         });
     }
+    if (utils.isLambdaEnv && !searchPaths.includes(LAMBDA_DEFAULT_NODE_MODULES_PATH)) {
```
Comment (haimrait) [inline src/events/module_utils.js:65]: this is the case if the user provides an external path to node_modules?

**Review by haimrait — APPROVED** [review]

File: src/events/module_utils.js (hunk 2 above)
Comment (ranrib) [reply src/events/module_utils.js:65]: This is for all cases, where we are running inside Lambda - to make sure we're looking at /var/task/node-modules

**Pull request merged**